lib/tasks/jenkins.rake

```diff
@@ -11,10 +10,10 @@ begin
 
     namespace :setup do
       task :pre_ci do
-        ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
-        gem 'ci_reporter'
+        require 'minitest/ci'
+        Minitest::Ci.report_dir = 'jenkins/reports/unit/'
       end
-      minitest_plugins = [:pre_ci, 'ci:setup:minitest']
+      minitest_plugins = [:pre_ci]
```

Comment:
Is it possible that katello enhances the `ci:setup:minitest` to do some weird custom stuff?
Find `ci:setup:minitest` didn't come up with anything, but there might be some codependency on task that Katello uses?